src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -184,7 +187,8 @@ export class IeObjectsController {
 		const relatedIeObjects = await this.ieObjectsService.getRelated(
 			schemaIdentifier,
 			meemooIdentifier,
-			referer
+			referer,
+			relatedQueryDto.maintainerId
```

Comment:
REQ: just pass the dto as full object to the service and extract maintainerId inside service.